src/components/TransactionsTab/helpers/getClearTransactionOptions.ts

```diff
@@ -0,0 +1,25 @@
+import i18n from "i18next";
+
+import { ClearOrderType } from "../../../types/clearOrderType";
+import { SelectOption } from "../../Dropdown/Dropdown";
+
+export default function getClearTransactionOptions(): SelectOption[] {
+  return [
+    {
+      label: i18n.t("common.all"),
+      value: ClearOrderType.all,
+    },
+    {
+      label: i18n.t("common.failed"),
+      value: ClearOrderType.failed,
+    },
+    // {
```

Comment:
I think we can remove these comments. I put in the last 2 options because they were included in the Figma design, but I don't think we have a practical use for them yet